src/compiler/checker.ts

```diff
@@ -133,9 +133,18 @@ namespace ts {
             signatureToString: (signature, enclosingDeclaration?, flags?, kind?) => {
                 return signatureToString(signature, getParseTreeNode(enclosingDeclaration), flags, kind);
             },
+            signatureToAccessibleString: (signature, enclosingDeclaration?, flags?, kind?) => {

```

Comment:
why do you need these on the checker? `getSymbolDisplayBuilder` is already visible on the `TypeChecker` interface. I would just move these to codefixs/helpers.